src/RelayBoardV3.cpp

```diff
@@ -130,13 +132,21 @@ void RelayBoardV3::handle(std::shared_ptr<const pilot::Incident> value){
 
 
 void RelayBoardV3::handle(std::shared_ptr<const pilot::SystemState> value){
+	if (value->system_errors.size() > 0) {
```

Comment:
Always do `m_system_error.clear()`. We fill it with the current list.

Reply:
:thinking: I have done it the line below..

Reply:
You mean if condition is not required?